crates/biome_js_analyze/src/aria_analyzers/a11y/no_redundant_roles.rs

```diff
@@ -94,7 +94,7 @@ impl Rule for NoRedundantRoles {
             rule_category!(),
             state.redundant_attribute_value.range(),
             markup! {
-                "Using the role attribute '"{role_attribute}"' on the '"{element}"' element is redundant."
+                "Using the role attribute '"{role_attribute}"' on the '"{element}"' element is redundant, since the role is already described by the semantic HTML element."
```

Comment:
We could simplify the diagnostic. Something like:

```suggestion
                "This role attribute '"{role_attribute}"' is redundant, because it is implied by the '"{element}"' element."
```

Reply:
I think the `semantic` part should still be there to let the dev know that this error will only occur in semantic HTML elements, not in stuff like `div`, what do you think?

Reply:
Ok, you can keep it :)